packages/atlas/src/components/_navigation/SidenavViewer/SidenavViewer.tsx

```diff
@@ -58,9 +59,11 @@ export const SidenavViewer: FC = () => {
   const [expanded, setExpanded] = useState(false)
   const followedChannels = usePersonalDataStore((state) => state.followedChannels)
   const unFollow = usePersonalDataStore((state) => state.actions.unfollowChannel)
+  const { activeMembership } = useUser()
   const {
     actions: { setAuthModalOpenName },
   } = useAuthStore()
+  const hasAtLeastOneChannel = activeMembership?.channels.length && activeMembership?.channels.length >= 1
```

Comment:
var naming assumes that it should be boolean
```suggestion
  const hasAtLeastOneChannel = !!activeMembership?.channels.length && activeMembership?.channels.length >= 1
```

current type is `boolean | 0 | undefined`